edit-post/components/visual-editor/style.scss

```diff
@@ -67,33 +67,30 @@
 }
 
 // The base width of the title should match that of blocks even if it isn't a block
-.edit-post-visual-editor .editor-post-title {
+.editor-post-title {
+	@include break-small() {
+		padding-left: $block-side-ui-padding;
+		padding-right: $block-side-ui-padding;
+	}
+}
+.edit-post-visual-editor .editor-post-title__block {
 	margin-left: auto;
 	margin-right: auto;
 	max-width: $content-width;
 
-	@include break-small() {
-		> div {
-			margin-left: -$block-side-ui-padding;
-			margin-right: -$block-side-ui-padding;
-		}
-	}
-
-
-	/*.editor-post-permalink {
-		left: $block-padding;
-		right: $block-padding;
-		color: $dark-gray-900;
+	// stack borders
+	> div {
+		margin-left: -1px;
```

Comment:
All these `1px` values are, I'm guessing, border-related as well? I definitely think it'd be worth having a `$border-size` or `$border-width` variable in that case.

Reply:
Good point. Considering #6773 which does major surgery on variables, I'd like to do that separately. Addressing the other bit now, thanks!